### ISSUE TITLE
Container fails to start in minikube

### ISSUE THREAD

**Issue by diversario** [bitnami/bitnami-docker-rabbitmq#46]
Pod log:
<details>

```
2017-04-06T23:26:29.950903826Z Welcome to the Bitnami rabbitmq container
2017-04-06T23:26:29.950912949Z Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-rabbitmq
2017-04-06T23:26:29.950922121Z Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-rabbitmq/issues
2017-04-06T23:26:29.950928247Z Send us your feedback at containers@bitnami.com
2017-04-06T23:26:29.952116830Z 
2017-04-06T23:26:33.429360051Z nami    INFO  Initializing rabbitmq
2017-04-06T23:26:33.481412903Z rabbitm INFO  Creating RabbitMQ System User...
2017-04-06T23:26:33.581137857Z rabbitm INFO  Setting RabbitMQ environment...
2017-04-06T23:26:33.619726450Z rabbitm INFO  Setting Erlang cookie...
2017-04-06T23:26:33.622379314Z rabbitm INFO  Managing RabbitMQ plugins...
2017-04-06T23:26:35.942056910Z rabbitm INFO  Configuring RabbitMQ permissions...
2017-04-06T23:26:35.997473464Z rabbitm INFO  Starting RabbitMQ node rabbit@localhost...
2017-04-06T23:26:43.938079195Z rabbitm INFO  Configuring RabbitMQ permissions...
2017-04-06T23:26:44.475586140Z rabbitm INFO 
2017-04-06T23:26:44.475936171Z rabbitm INFO  ########################################################################
2017-04-06T23:26:44.476131298Z rabbitm INFO   Installation parameters for rabbitmq:
2017-04-06T23:26:44.477042452Z rabbitm INFO     Username: user
2017-04-06T23:26:44.477242835Z rabbitm INFO     Node: rabbit@localhost
2017-04-06T23:26:44.477430608Z rabbitm INFO     Manager listening at: 0.0.0.0:15672
2017-04-06T23:26:44.477600358Z rabbitm INFO   (Passwords are not shown for security reasons)
2017-04-06T23:26:44.477840976Z rabbitm INFO  ########################################################################
2017-04-06T23:26:44.477991078Z rabbitm INFO 
2017-04-06T23:26:44.478198184Z nami    INFO  rabbitmq successfully initialized
2017-04-06T23:26:44.499623250Z INFO  ==> Starting rabbitmq...
2017-04-06T23:26:47.204304309Z File operation error: eacces. Target: /opt/bitnami/rabbitmq/var/lib/rabbitmq/mnesia/rabbit@localhost-plugins-expand/rabbitmq_management-3.6.9/ebin/dtls.beam. Function: get_file. Process: code_server.
2017-04-06T23:26:47.205418726Z 
2017-04-06T23:26:47.205699053Z =ERROR REPORT==== 6-Apr-2017::23:26:47 ===
2017-04-06T23:26:47.205759282Z File operation error: eacces. Target: /opt/bitnami/rabbitmq/var/lib/rabbitmq/mnesia/rabbit@localhost-plugins-expand/cowlib-1.0.2/ebin/dtls.beam. Function: get_file. Process: code_server.
2017-04-06T23:26:47.205805234Z 
2017-04-06T23:26:47.205848134Z =ERROR REPORT==== 6-Apr-2017::23:26:47 ===
2017-04-06T23:26:47.205890453Z File operation error: eacces. Target: /opt/bitnami/rabbitmq/var/lib/rabbitmq/mnesia/rabbit@localhost-plugins-expand/cowboy-1.0.4/ebin/dtls.beam. Function: get_file. Process: code_server.
2017-04-06T23:26:47.205945594Z 
2017-04-06T23:26:47.205987803Z =ERROR REPORT==== 6-Apr-2017::23:26:47 ===
2017-04-06T23:26:47.206046858Z File operation error: eacces. Target: /opt/bitnami/rabbitmq/var/lib/rabbitmq/mnesia/rabbit@localhost-plugins-expand/amqp_client-3.6.9/ebin/dtls.beam. Function: get_file. Process: code_server.
```
</details>

I'm installing this with these overrides in `values.yaml` (may not be correct)
```
rabbitmq:
  rabbitmqPassword: pass
  rabbitmqUsername: user
    
  service:
    spec:
    ports:
      - port: 15672
        name: http
        targetPort: 15672
```

`requirements.yaml`
```
  - name: rabbitmq
    version: 0.5.1
    repository: https://kubernetes-charts.storage.googleapis.com
```

Not sure why it's failing or how do fix this.

**Comment by sameersbn** [comment]
@diversario We are seeing this issue as well and trying to figure out the cause. So far we've not been able to isolate the issue, but we did find out that it has something to do with the persistent volumes as the chart works fine with persistence turned off.

**Comment by sameersbn** [comment]
@diversario Following up to notify you that we have found the issue and will be releasing a fix for this issue soon. Will keep up updated. 

Thanks for taking the time to report this issue.

**Comment by sameersbn** [comment]
Hello @diversario, please note that this issue was because of HostPath volume permissions in minikube which has been fixed in minikube 1.18.0. Requesting you retry after updating to the latest minikube release.

We hope you find this information useful. Do let us know it it resolves the issue. Thanks.

**Comment by sameersbn** [comment]
Closing issue. Please let us know if you need further assistance.
Join our slack community http://slack.oss.bitnami.com/. Hope to see you there  :slightly_smiling_face:
